litmus-portal/backend/graphql-server/graph/project.graphqls

```diff
@@ -0,0 +1,12 @@
+type Project{
+    id: ID!
+    name: String!
+    state: String
+    created_at: String!
+    updated_at: String!
+    removed_at: String!
+}
+
+input ProjectInput {
```

Comment:
Where is this type being used?

Reply:
Yes I dont think it is required now. Will remove it